CustomInvitationSample/CustomInvitationSample/foresee_configuration.json

```diff
@@ -0,0 +1,15 @@
+{
+    "clientId":"9NNs9GIBQIVdk1LpDlJpAQ==",
+    "notificationType":"IN_SESSION",
+    "measures":
+    [
+        {
+            "surveyId":"mobile_app-en",
+            "launchCount": 0
+        }
+    ],
+    "invite": {
+        "logo": "ForeSee_logo",
+        "baseColor": [237, 38, 54]
+    }
```

Comment:
I'm not sure if this is the right fix for this tbh... If the client choose to use a custom invite, why do they have to set the in-sdk invite logo and color? I feel we should update the sdk's warning message copies instead of updating the sample app.  What do you think @wayneburkett ?

Reply:
I agree, but that's an SDK change for another release. For now, I just want to show how to prevent the warnings. I'll make a new ticket to cover those changes.

Reply:
👍